source/applicability/basic_structure.rst

```diff
@@ -800,8 +800,8 @@ The following table is describing the variable content of the message.
    ============ ===============================
 
 
-Structure for status response message
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Structure for a status response message
```

Comment:
If we fix this, then we should also fix all the other titles as well, e.g. "Structure for alarm request message", "Structure for alarm acknowledgement message" and "Structure for alarm suspend message"

Reply:
Fixed with 07181d7e10454e1e2863c0a274674f7912a5aa6c